saw-script/src/SAWScript/REPL/Trie.hs

```diff
@@ -6,7 +6,16 @@ Maintainer  : huffman
 Stability   : provisional
 -}
 
-module SAWScript.REPL.Trie where
+module SAWScript.REPL.Trie (
+    Trie,
+    empty,
+    insert,
+    lookup,
+    lookupWithExact,
+    leaves
+ ) where
```

Comment:
It might be worth adding some module Haddocks where saying that `SAWScript.REPL.Trie` is intended to be imported qualified, given that it exports some commonly-used function names like `empty`, `insert`, and `lookup`.